war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -520,8 +520,10 @@ function makeButton(e,onclick) {
         btn.addListener("click",h);
     var be = btn.get("element");
     Element.addClassName(be,clsName);
-    if(n) // copy the name
+    be.setAttribute("title",e.value);
```

Comment:
This sets the tooltip on all buttons. Not sure whether that's intended. It's only really *needed* for the "X" close/delete buttons.